### ISSUE TITLE
Wrong required validation rule triggered

### ISSUE THREAD

**Issue by Kaos1337** [PH7-Jack/livewire-ui#138]
**Describe the bug**
Hi,
I'm encountering a weird behaviour having more than one select on the same page. The issue is with the required validation rule of the select components: if I change the option of the second select, the first one select has the validation triggered and seems that the first one hasn't a choosen option (even if it has an option selected).

**To Reproduce**
Go to the page with the two selects, change only the second one and see the first one validation rule triggered.

**Expected behavior**
Since that I never changed the first select, its validation rule shouldn't be triggered.

**Screenshots**
https://i.imgur.com/mCSu8EY.png
https://i.imgur.com/wkE7wc4.png

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Google Chrome
 - Version 96

```
<x-form.label name="Quality" class="mt-6"/>
        <x-select
            placeholder="Select a quality"
            :options="[
                ['name' => 'Low',       'id' => 1],
                ['name' => 'Medium',       'id' => 2],
                ['name' => 'Good',       'id' => 3],
                ['name' => 'High',        'id' => 4],
                ['name' => 'Excellent',  'id' => 5]
            ]"
            option-label="name"
            option-value="id"
            wire:model="quality"
        />

        <x-form.label name="Object" class="mt-6"/>
        <x-select
            placeholder="Choose an object"
            :options="[
                ['name' => 'Object 1', 'id' => 1],
                ['name' => 'Object 2', 'id' => 2]
            ]"
            option-label="name"
            option-value="id"
            wire:model="object"
        />
```

```
protected $rules = [
        'quality' => 'required|integer|min:1|max:5',
        'object' => 'required|integer|min:1|max:2'
    ];


// this is just to set the default value
public function mount(...)
    {
...
        $this->quality = ...->quality;
        $this->object = ...->object;
    }

    public function updated($propertyName)
    {
        $this->validateOnly($propertyName);
    }
```

**Comment by ph7jack** [comment]
Hello @Kaos1337, can you explain more detail this issue? I simulate here and is ok
Record a video if need
https://www.loom.com/share/f683e33aabd14faa99197debd8d22ae2

**Comment by ph7jack** [comment]
What are your livewire and alpine version?

**Comment by Kaos1337** [comment]
Livewire 2.8.0 and alpine 3.5.2.
I found a workaround that helps to focus on the source of the issue. Try to give the name of the first select: "aaa_bbb_ccc" and the name of the second select "aaa_bbb".
So:
```
<x-select
...
wire:model="aaa_bbb_ccc"
/>

<x-select
...
wire:model="aaa_bbb"
/>
```
and in the rules (`protected $rules = [...`) too, etc.
To simplify the example I changed them but the real case is: channel_codec_quality and channel_codec.
I don't understand why with the second name that casually has the root of the first one cause the issue. Giving them 2 distinct names avoid this situation.
